src/popup/views/app.vue

```diff
@@ -13,17 +13,11 @@
         <span v-text="i18n('menuDashboard')"></span>
       </div>
     </div>
-    <div class="menu menu-domains" v-show="domains.length" :class="{expand: activeMenu === 'domains'}">
-      <div class="menu-item" @click="onClickDomains">
+    <div class="menu">
```

Comment:
`v-show` should be kept here

Reply:
Should it be now `v-show="domain"` (because now `domains` array is replaces by `domain` string)?

Reply:
I think it should be `v-show="store.domain"` now